src/app/interfaces/enclosure-tray.interface.ts

```diff
@@ -0,0 +1,7 @@
+export interface TrayConfig {
```

Comment:
If this is specific to enclosure module, you can keep it in the `enclosure` folder.